### PR TITLE
port details for APIServerURL

### PR DESCRIPTION
**Pull request by garbagego**
#https://github.com/openshift/cluster-kube-apiserver-operator/pull/855#

### CODE REVIEW

File: config/v1/types_infrastructure.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ type InfrastructureStatus struct {
 	EtcdDiscoveryDomain string `json:"etcdDiscoveryDomain"`
 
 	// apiServerURL is a valid URI with scheme(http/https), address and
-	// port.  apiServerURL can be used by components like the web console
+	// port(only if not using default port HTTP 80 and HTTP/TLS 443).  apiServerURL can be used by components like the web console
```
Comment (sttts) [inline config/v1/types_infrastructure.go:70]: How about `... and optionally a port (defaulting to 80 for http and 443 for https)`?

You also have to generate the CRDs. `make update` or something like that. `make help` shows you the commands.

Comment (garbagego) [reply config/v1/types_infrastructure.go]: @sttts  Thanks for your suggestion.
I have updated the api doc and updated CRD by calling `make update-codegen-crds`

File: config/v1/0000_10_config-operator_01_infrastructure.crd.yaml [hunk 2]
```diff
@@ -158,7 +158,8 @@ spec:
               type: string
             apiServerURL:
               description: apiServerURL is a valid URI with scheme(http/https), address
-                and port.  apiServerURL can be used by components like the web console
+                and port and optionally a port (defaulting to 80 for http and 443
```
Comment (sttts) [inline config/v1/0000_10_config-operator_01_infrastructure.crd.yaml:161]: only one "and port"

**Comment by sttts** [comment]
Need a swagger doc update.

**Pull request merged**